core/src/main/java/org/bitcoinj/wallet/WalletProtobufSerializer.java

```diff
@@ -707,17 +707,16 @@ private WalletTransaction connectTransactionOutputs(final NetworkParameters para
         final WalletTransaction.Pool pool;
         switch (txProto.getPool()) {
             case DEAD: pool = WalletTransaction.Pool.DEAD; break;
-            case PENDING: pool = WalletTransaction.Pool.PENDING; break;
```

Comment:
I think for clarity it's better to keep the two deprecated values at the bottom, even if it's duplicating one statement.

We could maybe think about removing the values though, I'm not sure.